backend/tests/test_add_item.py

```diff
@@ -89,6 +88,102 @@ def test_add_new_item_with_image(client):
     assert new_price.downvotes == []
 
```

Comment:
You do not seem to have a test case for when an item is added without an image, and then later, the user wants to add an image to that existing item.

Reply:
That's because that is impossible in the current implementation; once an item is added, an image cannot be uploaded later. Uploading an image for an existing item would be a separate user story. So we don't test this case in this user story.